disnake/channel.py

```diff
@@ -760,7 +762,9 @@ async def create_thread(
         name: str,
         message: Optional[Snowflake] = None,
         auto_archive_duration: AnyThreadArchiveDuration = None,
-        type: Optional[ChannelType] = None,
+        type: Optional[
+            Literal[ChannelType.public_thread, ChannelType.private_thread, ChannelType.news_thread]
```

Comment:
This and the overload above could use `ThreadType` defined in threads.py

Reply:
https://github.com/DisnakeDev/disnake/pull/543/commits/234936ef4fc6725bbf607c02357419c20e99716b